### DISCUSSION TITLE
Send helpop to other servers trough CMIBungee

### DISCUSSION THREAD

**Discussion by DaveDevil** [Zrips/CMI#3464] [Feature Requests]
**Description of feature/improvement request:**

Add an option to enable the broadcasting of an helpop message to all other servers in a bungeecord network

---
### Detailed explanation of what it should do, when and in what situation: ###
Broadcasting of an helpop message to all other servers in a bungeecord network when a player does /cmi helpop <text here>

### Basic explanation as to why this feature is needed and/or useful to have: ###

If a staff member is not in the same server of the player who's asking for help, this is needed in the same way that even the staffchat works across multiple servers.

**Comment by Gayytis** [comment]
That would be very cool, indeed !
